### ISSUE TITLE
Oxford2 dataset ground truths

### ISSUE THREAD

**Issue by Idanlau** [tiev-tongji/Spatio-Temporal-SeqVPR#6]
Hey @SunSongMu @kaiyi98  thanks for the great work!

I’d like to clarify the CSV files used to establish the Oxford2 dataset. It appears that for the Oxford2 training set—both the queries and the database—gps.csv was used to assign locations to each frame, as rtk.csv was unavailable for the traversals. Additionally, I noticed a discrepancy in the reported number of training query images. While your paper lists 3,926 images, I obtained 3,296 images. Since the setup appears to follow the methodology from [Learning Sequential Descriptors for Sequence-based Visual Place Recognition](https://arxiv.org/pdf/2207.03868), which reports around 3.3k images for the Oxford training queries, my results seem to align more closely with that figure. I have attached screenshots from your paper and their paper below.

For the test set, it seems that gps.csv was used for the test database (even though rtk.csv was available but did not match the expected image count from the paper), whereas rtk.csv was employed for the test queries.

<img width="505" alt="Image" src="https://github.com/user-attachments/assets/c7a82ee7-1126-48d5-a45e-5d2f41521022" />

<img width="687" alt="Image" src="https://github.com/user-attachments/assets/5b90309b-6723-4956-8259-3b250b4aa64c" />
